### ISSUE TITLE
v0.1 Onboarding

### ISSUE THREAD

**Issue by rase-rocks** [Co-Epi/app-ios#13]
I'm creating this issue so it can be assigned to myself. Sorry if this is not the place ;)

**Comment by rase-rocks** [comment]
![Simulator Screen Shot - iPhone 11 Pro Max - 2020-03-27 at 10 43 38](https://user-images.githubusercontent.com/38786434/77748334-16119b00-7018-11ea-94a2-1c63d6c95fae.png)

First, and incomplete draft. I have created an AttributedString class so we can build up translatable NSAttributedStrings for use in UILabels. It means we can follow the spec with bold and normal text combined. It works with left to right languages, but without knowledge of any I'm not sure how it will go with right to left languages. In other words the bold words might get mixed up in translation.

Also all the strings have been moved into a separate struct for ease of translation later. They are all NSLocalizableStrings.
